virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
```

Comment:
I'm not sure about "at version ...". Your call, but I think I would have phrased it, "{VirtProductName} version 2.4.z or earlier."
Re: "z" version. It might be clearer to provide the actual number for the version, for example, "2.4.6 or earlier".